src/components/Article.jsx

```diff
@@ -1 +1,34 @@
-// export const Article = () => ();
+import React from "react";
+
+export const Article = ({
+  title,
+  date,
+  text,
+}) => {
+  const dateISO = date.toISOString().slice(0, 10);
+  const dateLocaleString = date.toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  });
+
+  return (
+    <>
```

Comment:
this fragment is unnecessary as you have only 1 child element

Reply:
done